utils/autotailor

```diff
@@ -198,27 +247,66 @@ class Tailoring:
         title.text = self.profile_title
 
         self._add_rule_select_operations(profile)
-        self._add_value_selections(profile)
+        self._add_value_overrides(profile)
         self.rule_refinements_to_xml(profile)
+        self.value_refinements_to_xml(profile)
 
         root_str = ET.tostring(root)
         pretty_xml = xml.dom.minidom.parseString(root_str).toprettyxml()
         with open(location, "w") if location != "-" else sys.stdout as f:
             f.write(pretty_xml)
 
+    def import_json_tailoring(self, json_tailoring):
+        import json
+        with open(json_tailoring, "r") as jf:
+            all_tailorings = json.load(jf)
+
+        if 'profiles' in all_tailorings and all_tailorings['profiles']:
+            # We currently support tailoring of one profile only
+            tailoring = all_tailorings['profiles'][0]
+        else:
+            raise ValueError("JSON Tailoring does not define any profiles.")
+
+        self.extends = tailoring["base_profile_id"]
+
+        self.profile_id = tailoring.get("id", self.profile_id)
+        self.profile_title = tailoring.get("title", self.profile_title)
 
-def parse_args():
+        if "rules" in tailoring:
+            for rule_id, props in tailoring["rules"].items():
+                if "evaluate" in props:
+                    if props["evaluate"]:
+                        self.rules_to_select.append(rule_id)
+                    else:
+                        self.rules_to_unselect.append(rule_id)
+                for attr in ATTRIBUTES:
+                    if attr in props:
+                        self.change_rule_attribute(rule_id, attr, props[attr])
+
+        if "variables" in tailoring:
+            for variable_id, props in tailoring["variables"].items():
+                if "value" in props:
+                    self.add_value_change(variable_id, props["value"])
+                if "option_id" in props:
+                    self.change_value_attribute(variable_id, "selector", props["option_id"])
+
+
+def get_parser():
     parser = argparse.ArgumentParser(
         description="This script produces XCCDF 1.2 tailoring files "
         "to be used by SCAP scanners and SCAP data streams.")
     parser.add_argument(
         "datastream", metavar="DS_FILENAME",
         help="The tailored data stream filename.")
     parser.add_argument(
-        "profile", metavar="BASE_PROFILE_ID",
+        "profile", metavar="BASE_PROFILE_ID", nargs='?', default="",
         help="Specify ID of the base profile. ID of the profile can be "
         "either its full ID, or the suffix, in which case the "
         "'xccdf_<id-namespace>_profile' prefix will be prepended internally.")
+    parser.add_argument(
+        "--json-tailoring", metavar="JSON_TAILORING_FILENAME", default="",
```

Comment:
What is your opinion on adding a short variant of this option, for example `-j`? I think that would simplify and make the usage of the tool more comfortable.